test/Polar/hessian.jl

```diff
@@ -12,17 +14,22 @@ import ExaPF: PowerSystem, AutoDiff
 
 const PS = PowerSystem
 
-# Warning: currently works only on CPU, as depends on
-# explicit evaluation of Hessian, using MATPOWER expressions
 @testset "Compute reduced Hessian on CPU" begin
     @testset "Case $case" for case in ["case9.m", "case30.m"]
+        if has_cuda_gpu()
+            ITERATORS = zip([CPU(), CUDADevice()], [Vector, CuVector])
+        else
+            ITERATORS = zip([CPU()], [Vector])
+        end
+        @testset "Device $device" for (device, T) in ITERATORS
```

Comment:
Could be indent the lines inside `@testset`?

Reply:
Alright! Done.